README.md

```diff
@@ -1,9 +1,3 @@
-![Version](https://img.shields.io/badge/Version-3.5.4-orange)
-[![GitHub issues](https://img.shields.io/github/issues/cheeeeeeeeeen/RoR2-ChensGradiusMod)](https://github.com/cheeeeeeeeeen/RoR2-ChensGradiusMod/issues)
-[![GitHub pull requests](https://img.shields.io/github/issues-pr/cheeeeeeeeeen/RoR2-ChensGradiusMod)](https://github.com/cheeeeeeeeeen/RoR2-ChensGradiusMod/pulls)
-[![Support Chen](https://img.shields.io/badge/Support-Chen-ff69b4)](https://ko-fi.com/cheeeeeeeeeen)
-![Maintenance Status](https://img.shields.io/badge/Maintenance-Inactive-orange)
-
```

Comment:
You can actually just change the version number here and the badges will still work. Probably just remove the Support Chen and PR ones.

Reply:
latest commit should resolve this, also added Inspect mode support that was introduced in the new DLC

Reply:
What's inspect mode again?

Reply:
Ah nvm. A simple search did it. Alright.